_docs/concepts/what-is-grafeas/overview.md

```diff
@@ -1,223 +1,111 @@
 
-**Occurrences**: An occurrence can be thought of as an instantiation of a note and describes how the note was found in a specific cloud resource or project (e.g., location, specific remediation steps, etc.), or what the results of a specific note were (e.g., the container images that resulted from a build). For example, an occurrence might report that the heartbleed OpenSSL bug (a possible Note) was found in a specific package of a container image, and include information about how to remedy the heartbleed bug based on the customer’s package.
-
-An occurrence name should take the format `/projects/<project_id>/occurrences/<occurrence_id>` where the project_id would typically be different from the project where the note is created and the occurrence_id would be unique per occurrence-project, and would often be random. 
+## Definition of terms
 
-Write access to occurrences should only be granted to users who have access to link a note to the occurrence. Any users can have read access to occurrences. 
+### Notes
+
+A _note_ describes a high-level piece of metadata. Notes are generally found via
+analysis and occur multiple times across different projects. For example, you
+could create a note about a particular vulnerability after analyzing a Linux
+package. You would also use a note to store information about the builder of a
+build process. Notes are often owned and created by the providers doing the
+analysis.
+
+Note names should follow the format `/projects/<project_id>/notes/<note_id>`.
+The note ID must be unique per project, and as informative as possible. For
+example, the name of a vulnerability note could be `CVE-2013-4869`, referencing
+the CVE it describes.
+
+It's generally preferable to store notes and occurrences in separate projects,
+allowing for more fine-grained access control.
+
+Notes should be editable only by the note owner, and read-only for users who
+have access to occurrences referencing them.
+
+### Occurrences
+
+An _occurrence_ is essentially an instantiation of a note. Occurrences describe
+project-specific details of a given note. For example, an occurrence of a note
+about a vulnerability would describe the package that the vulnerability was
+found in, specific remediation steps, and so on. Alternatively, an occurrence of
+a note about build details would describe the container images that resulted
+from a build.
+
+Occurrence names should follow the format
+`/projects/<project_id>/occurrences/<occurrence_id>`. The occurrence ID must be
+unique per project and is often random. Typically, occurrences are stored in
+separate projects than those where notes are created.
+
+Write access to occurrences should only be granted to users who have access to
+link a note to the occurrence. Any users can have read access to occurrences.
+
+### Resource URLs
+
+A _resource URL_ is a unique URL for the resource to which a given occurrence
+applies. Common examples of resources are containers, Virtual Machine (VM)
```

Comment:
should this be "containers" or "container images"?

Reply:
I think container images -- changed it.